src/views/App/index.tsx

```diff
@@ -52,6 +53,7 @@ const AppRoutes = () => {
       <Redirect exact from="/login" to="/signin" />
       <Route path="/signin" exact component={Login} />
       <SecureRoute path="/user-diagnostics" component={UserInfo} />
+      <SecureRoute path="/my-requests" component={MyRequests} />
```

Comment:
If this isn't ready to be viewed in the app, can we feature flag it?

Reply:
I think this is ok? It's hidden through obfuscation, which is a pattern we've used before.